requirements.txt

```diff
@@ -1 +1 @@
-hvac>=0.10.6
+hvac>=1.1.0
```

Comment:
Does it make sense to add a version interval to benefit from some of the updates, but prevent the potential upstream breaking changes?